auth/postgres/domains.go

```diff
@@ -96,6 +96,36 @@ func (repo domainRepo) RetrieveByID(ctx context.Context, id string) (auth.Domain
 	return auth.Domain{}, repoerr.ErrNotFound
 }
 
+func (repo domainRepo) RetrieveBasicInfoByID(ctx context.Context, id string) (auth.Domain, error) {
```

Comment:
Why do we need a new repo method, i.e. why don't we use `RetrieveByID`?

Reply:
Because RetrieveByID returns more info than we need. Do you propose we modify the existing method to perform both functions instead?

Reply:
Also, having this dedicated method is self documenting so it's easy to understand it's purpose on the service layer.